apps/api/v2/src/modules/auth/guards/organizations/is-org.guard.ts

```diff
@@ -0,0 +1,28 @@
+import { OrganizationsRepository } from "@/modules/organizations/organizations.repository";
+import { Injectable, CanActivate, ExecutionContext, ForbiddenException } from "@nestjs/common";
+import { Request } from "express";
+
+import { Team } from "@calcom/prisma/client";
+
+@Injectable()
+export class isOrgGuard implements CanActivate {
+  constructor(private organizationsRepository: OrganizationsRepository) {}
+
+  async canActivate(context: ExecutionContext): Promise<boolean> {
+    const request = context.switchToHttp().getRequest<Request & { organization: Team }>();
+    const organizationId: string = request.params.orgId || request.params.organizationId;
```

Comment:
Why do we allow passing ":orgId" and ":organizationId" instead of just either of them?